### DISCUSSION TITLE
Can't do deploy

### DISCUSSION THREAD

**Discussion by proxima812** [vercel/community#3702] [Help]
### Page to Investigate

https://vercel.com/proxima812/12-infinity/EAWXj3JPNHiBiW6fMFNudWPTpnjH

### Steps to Reproduce

Running build in Washington, D.C., USA (East) – iad1
--
23:30:09.681 | Cloning github.com/proxima812/12-infinity (Branch: main, Commit: da8286b)
23:30:09.687 | Skipping build cache, deployment was triggered without cache.

and so it can go for a couple of hours...

**Comment by amyegan** [comment]
Please share your [Build and Development Settings](https://vercel.com/docs/deployments/configure-a-build#build-and-development-settings) and a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) see we can see what is happening 🙏

**Reply by proxima812** [reply]
![Screenshot (2)](https://github.com/vercel/community/assets/116840531/a7d9dcf7-c7d3-4e90-a957-5e05a8a030a8)

**Reply by amyegan** [reply]
Thanks for sharing the build settings. Please also share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) see we can see what happens when building the site

**Reply by proxima812** [reply]
The problem is that there are these lines in the build logs:

Running build in Washington, D.C., USA (East) – iad1
Cloning github.com/proxima812/12-infinity (Branch: main, Commit: da8286b)
Skipping build cache, deployment was triggered without cache.

Sometimes without: "Skipping build cache, deployment was triggered without cache."

Then nothing happens. That is, it will not offend files and so on.

I'm sorry, maybe I don't understand.
